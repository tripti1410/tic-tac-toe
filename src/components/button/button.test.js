import React from "react";
import renderer from "react-test-renderer";
import Button from "./button";

it("renders correctly - default state", () => {
  const tree = renderer.create(<Button buttonLable="" />).toJSON();
  expect(tree).toMatchSnapshot();
});
