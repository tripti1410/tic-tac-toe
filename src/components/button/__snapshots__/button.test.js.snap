// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly - default state 1`] = `
<button
  className="reset-button"
  role="button"
/>
`;

exports[`test renders correctly - default state 1`] = `
<button
  className="reset-button"
  onClick={undefined}
  role="button" />
`;
